lib/api/web-element/commands/isSelected.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Determines if an element is selected.
+ *
+ * For more info on working with DOM elements in Nightwatch, refer to the <a href="https://nightwatchjs.org/guide/writing-tests/finding-interacting-with-dom-elements.html">Finding & interacting with DOM Elements</a> guide page.
+ *
+ * @example
+ * describe('isSelected Demo', function() {
+ *
+ *   it('test isSelected', function(browser) {
+ *     browser
+ *       .element('#search')
+ *       .isSelected(function callback(result) {
+ *         assert.equal(result.value, true);
+ *       });
```

Comment:
`isSelected()` command does not support a callback. Also, `.element` should come in first line itself because `isSelected()` is a call on `browser.element()` and not just `browser`.

```suggestion
 *     browser.element('#search')
 *       .isSelected()
 *       .assert.equals(true);
```